src/test/data/datasets.py

```diff
@@ -0,0 +1,128 @@
+import os
```

Comment:
Is the idea here to set up fixture data more easily with pytest? Probably just me not seeing how this is linked to the java client test